huaweicloud/services/meeting/resource_huaweicloud_meeting_conferences.go

```diff
@@ -0,0 +1,1111 @@
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validation.StringLenBetween(0, 96),
+						},
+						"role": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.IntInSlice([]int{
+								int(RoleTypeGuest), int(RoleTypeHost),
+							}),
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(AccessTypeNormal), string(AccessTypeTelepresence), string(AccessTypeTerminal),
+								string(AccessTypeOutside), string(AccessTypeMobile), string(AccessTypeTelephone),
+								string(AccessTypeIdeahub),
+							}, false),
+						},
+						"is_mute": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.IntInSlice([]int{
+								int(MuteTypeOff), int(MuteTypeOn),
+							}),
+						},
+						"is_auto_invite": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.IntInSlice([]int{
+								int(InviteTypeAuto), int(InviteTypeManual),
+							}),
+						},
+						"phone": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"email": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+						"sms": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"cycle_params": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cycle": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(CycleTypeDay), string(CycleTypeWeek), string(CycleTypeMonth),
+							}, false),
+						},
+						"pre_remind": {
+							Type:         schema.TypeInt,
+							Required:     true,
+							ValidateFunc: validation.IntBetween(0, 30),
+						},
+						"start_date": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringMatch(regexp.MustCompile(`^\d{4}-\d{2}-\d{2}$`),
+								"The date format should be 'YYYY-MM-DD'."),
+						},
+						"end_date": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.StringMatch(regexp.MustCompile(`^\d{4}-\d{2}-\d{2}$`),
+								"The date format should be 'YYYY-MM-DD'."),
+						},
+						"interval": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+						},
+						"points": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeInt},
+						},
+					},
+				},
+			},
+			"configuration": {
```

Comment:
all parameters in this structure are optional, is it ok if the user input an empty structure?

Reply:
ditto